examples/wagmi/src/index.tsx

```diff
@@ -2,17 +2,16 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom/client';
 
 import { mainnet, goerli } from 'viem/chains';
-import { WagmiConfig, createConfig, configureChains, Connector } from 'wagmi';
-import { publicProvider } from 'wagmi/providers/public';
-import { SafeConnector } from 'wagmi/connectors/safe';
-import { InjectedConnector } from 'wagmi/connectors/injected';
-import { MetaMaskConnector } from 'wagmi/connectors/metaMask';
-import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
-
+import { WagmiProvider, createConfig, CreateConnectorFn, http } from 'wagmi';
 import { Buffer } from 'buffer';
+import { metaMask } from 'wagmi/connectors';
```

Comment:
I was wondering about this. They say they removed MetaMaskConnector but they still have this connector available.

https://wagmi.sh/react/guides/migrate-from-v1-to-v2#removed-metamaskconnector